typings/environment.d.ts

```diff
@@ -23,8 +23,8 @@ export interface Environment extends DefaultElements<EnvironmentProps>, MetaSys<
   createAssetWithId(id: string, data: AssetProps): Promise<Asset>,
   createContentType(data: ContentTypeProps): Promise<ContentType>,
   createContentTypeWithId(id: string, data: ContentTypeProps): Promise<ContentType>,
-  createEntry(contentTypeId: string, data: EntryProp): Promise<Entry>,
-  createEntryWithId(contentTypeId: string, id: string, data: EntryProp): Promise<Entry>,
+  createEntry<TFieldType = any>(contentTypeId: string, data: EntryProp): Promise<Entry<TFieldType>>,
```

Comment:
Those `= any` will override the defaults of the `KeyValueMap` type if not specified.
Probably should put the `KeyValueMap` as a default instead.